src/site/paragraphs/q_a_section.drupal.liquid

```diff
@@ -21,6 +21,7 @@
     {% include "src/site/paragraphs/q_a.collapsible_panel.drupal.liquid" questions = entity.fieldQuestions %}
 {% else %}
     {% for question in entity.fieldQuestions %}
-      {% include "src/site/paragraphs/q_a.drupal.liquid" paragraph = question.entity sectionHeader = entity.fieldSectionHeader %}
+      {% assign sectionHeader = entity.fieldSectionHeader %}
```

Comment:
this needs to be assigned before we use it in the include so that we can keep the templates consistent and not have one-off `if` statements for template includes